keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -774,26 +751,19 @@ are missing a bunch of machinery and tooling and can't do that now.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-<!--
-Even if applying deprecation policies, they may still surprise some users.
--->
+No
 
 ### Monitoring Requirements
 
-<!--
-This section must be completed when targeting beta to a release.
-
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
--->
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-<!--
-Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-checking if there are objects with field X set) may be a last resort. Avoid
-logs or events for this purpose.
--->
+A new group of metrics are added for the new Cluster IP allocators
+1. IP allocator pkg/registry/core/service/ipallocator/metrics.go
```

Comment:
These paths aren't super helpful, the go code is going to move around, can we list out the key metric names? Also can we explain how an operator would actually use these? Assume the operator doesn't read go but does run Kubernetes clusters and has never looked at these metrics before. How can they look at them to make sure this upgrade went safely?